docs/product/metrics/metrics-examples.mdx

```diff
@@ -0,0 +1,35 @@
+| Number of user signups                       | counter            | Sentry.metrics.increment(”signups”, 1, { tags: { plan_type: “freemium”}});                                                              |
+| Measure UX improvement to signup form        | distribution       | Sentry.metrics.distribution("login.time_to_finish_form", 14.3 { tags: { subpage: “signup”}, unit: “second”});                           |
+| Track the loading times for a component      | distribution       | Sentry.metrics.distribution("component_load_time", 15.0, {tags: { type: "important" },unit: "millisecond"});                            |
+| Track API request payloads                   | distribution       | Sentry.metrics.distribution("api_request_payload", 136.3, { tags: { api_name: “maps_provider”, subpage: “signup”}, unit: “kilobyte”});  |
+| Count concurrent users in application        | gauge              | Sentry.metrics.gauge("concurrent_users", 10, {  tags: { browser: "Firefox", app_version: "1.0.0"}});                                    |
+| Track the number of items in a shopping cart | gauge              | Sentry.metrics.gauge("shopping_cart_item_count", itemCount, { tags: { platform: "web"}});                                               |
+| Count the unique number of users on a page   | set                | Sentry.metrics.set("user_view", "jane", tags: {"page": "/home"});                                                                       |
+| Count unique number of user sessions         | set                | Sentry.metrics.set("unique_active_sessions", sessionId, { tags: { platform: "mobile"}});                                                |
+
+
+## Backend Examples
+
+| **Use Case**                                       | **Metric Type**    | **Example**                                                                                                                                           |
+| -------------------------------------------------- | -------------------| ----------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Count of API calls of a 3rd party API              | counter            | sentry_sdk.metrics.incr(key="api_calls", value=1, tags={"api": "third_party"})                                                                        |
+| Count of jobs executed per worker                  | counter            | sentry_sdk.metrics.incr(key=”external_api_calls”, value=1,<br> tags={”worker_id”: “485ea324-003d-4f30-a0f9-beed9792c4a1”, “worker_type”: ”standard”}) |
+| Processing time for a task	                       | distribution       | sentry_sdk.metrics.distribution(key=”processing_time”, value=0.002, unit=”second”, tags={”task”: “example_task”})                                     |
```

Comment:
Could this tab in the first column be the problem?

Reply:
Unfortunately it wasn't it 😢